src/components/welcome-screen/welcome-screen.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+  const WelcomeScreen = (props) => {
+  const {errorsCount} = props;
```

Comment:
используй сокращенную форму для стрелочных функций

```suggestion
  const WelcomeScreen = ({ errorsCount }) => (
  <section ...>
```